doc/languages-frameworks/python.section.md

```diff
@@ -1061,11 +1061,11 @@ in pkgs.mkShell rec {
     pythonPackages.numpy
     pythonPackages.requests
 
-    # the following packages are related to the dependencies of your python
+    # The following packages exemplify dependencies of your python
```

Comment:
I'm not a native english speaker, I may be wrong but this sentece feels awfully long. Can you consider adding some punctuation or breaking it up?

Reply:
Since it's rather hard to estimate how someone reads it, a suggestion would be helpful. Personally I don't think the sentence is confusing (I didn't write the original though, just tried minimally modifying it to avoid some confusion I've seen).

Would sth like

```
# The following packages exemplify dependencies of your python
# project.
# In this particular example the python modules listed in the
# hypothetical requirements.txt require the following packages
# to compile binary extensions.
```

be easier? It's at least shorter and imho does not lose much content (assuming the user understands what native dependencies for python packages are).

Reply:
Someone suggested me to put it like this. What do you think?
```
In this particular example, in order to compile any binary extensions they may
require, the python modules listed in the hypothetical requirements.txt need
the following packages to be installed locally:
```
It keeps the original content but it feels tidier to me.

Reply:
honestly, the whole line could be removed, we are repeating ourselves the next line anyway.

Reply:
@jonringer: I assumed it was more about the second line than the first one, since the first one is not exactly a complex sentence, but let's take @rnhmjoj suggestion which drops it anyways.